src/DependencyInjection/Configuration.php

```diff
@@ -34,6 +34,9 @@ public function getConfigTreeBuilder()
                     ->isRequired()
                     ->info('The path where Encore is building the assets - i.e. Encore.setOutputPath()')
                 ->end()
+                ->scalarNode('crossorigin')
```

Comment:
as there is only 2 valid values, I suggest using an `enumNode` instead

Reply:
Done.